extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -169,6 +174,9 @@ export class AmpStoryInteractive extends AMP.BaseElement {
     /** @protected {?Element} */
     this.rootEl_ = null;
 
+    /** @public {../../../src/service/localizationService} */
+    this.localizationService = Services.localizationForDoc(this.element);
```

Comment:
This is async and might not be ready when the page loads. You might need to get this in the buildcallback before it's resolved.
<img width="895" alt="Screen Shot 2020-12-17 at 10 59 19 AM" src="https://user-images.githubusercontent.com/3860311/102511678-11ce5780-4057-11eb-9d53-05b6e9ec4a9f.png">

Example:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-story-360/0.1/amp-story-360.js#L402-L407